GomokuClient/packages/gomoku-core/src/hooks/useBoard.ts

```diff
@@ -40,11 +46,14 @@ export const useBoard = () => {
     if (value !== null) return;
     lastRow.current = row;
     lastCol.current = col;
-    updateBoard(row, col, isBlackNext.current ? "black" : "white");
+    updateBoard(row, col, isBlackNext.current ? "Black" : "White");
     isBlackNext.current = !isBlackNext.current;
   };
 
-  const setOpponentMove = useCallback(
+  // TODO: Currently function named with 'Piece' word as it was previously in this code. My suggestion is to align names how call stuff in FE and BE
+  // TODO: In backend I call one piece tile. Coordinates are x and y. For better and cleaner solution let's align on namings
+  // TODO: We can even write documentation and figure out how we call things.
```

Comment:
That's a good catch and mindset of thinking, let's align on this on the call then. Maybe we can configure linting for specific methohd to be tide to the hook classes

Reply:
Ok, I will create issue and remove comments from code

Reply:
https://github.com/ligomoku/gomoku/issues/121